testing/mbuffer/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Katie Holly <holly@fuslvz.ws>
+# Maintainer: Katie Holly <holly@fuslvz.ws>
+pkgname=mbuffer
+pkgver=20171011
+pkgrel=0
+pkgdesc="mbuffer buffers I/O operations and displays the throughput rate"
+url="http://www.maier-komor.de/mbuffer.html"
+arch="all"
+license="GPL-3.0"
+depends=""
+makedepends="linux-headers libressl-dev"
+install=""
+subpackages="$pkgname-doc"
+source="
+	http://www.maier-komor.de/software/mbuffer/mbuffer-$pkgver.tgz
+	"
```

Comment:
When there is only a single url we normally have it on a single line:
`source="http://www.maier-komor.de/software/mbuffer/mbuffer-$pkgver.tgz"`